lib/player_input.rb

```diff
@@ -0,0 +1,36 @@
+require_relative "coordinates"
+require_relative "null_coordinates"
+
+class PlayerInput
```

Comment:
This is verbatim what `PlayerTurn` used to be.

Reply:
👍